packages/launch-config/package.json

```diff
@@ -34,6 +34,9 @@
         "@sap-ux/project-access": "workspace:*",
         "@sap-ux/ui5-config": "workspace:*",
         "@sap-ux/ui5-info": "workspace:*",
+        "@sap-ux/odata-service-inquirer": "workspace:*",
```

Comment:
Please remove this dependency, this is using types not intended for this purpose and should only be consumed if you are calling `getPrompts` or `prompt`